File: apps/cargo-tracker/src/main/java/com/example/cargotracker/estimation/application/CreateEstimateCommand.java
package com.example.cargotracker.estimation.application;

import java.math.BigDecimal;
import java.time.LocalDate;

public record CreateEstimateCommand(
        String originUnlocode,
        String destinationUnlocode,
        LocalDate arrivalDeadline,
        String cargoType,
        BigDecimal weightKg
) {}

File: docs/article/source/csharp/apps/cargo-tracker/src/CargoTracker.Web/Estimation/Application/Internal/CommandServices/CreateEstimateCommand.cs
using CargoTracker.Estimation.Domain.Model;

namespace CargoTracker.Estimation.Application.Internal.CommandServices;

/// <summary>輸送見積作成コマンド（US01）。</summary>
public sealed record CreateEstimateCommand(
    string OriginUnLocode,
    string DestinationUnLocode,
    DateOnly ArrivalDeadline,
    CargoType CargoType,
    decimal WeightKg);
